packages/spor-react/src/theme/slot-recipes/tabs.ts

```diff
@@ -98,11 +154,15 @@ export const tabsSlotRecipe = defineSlotRecipe({
       lg: {
         list: {
           height: 8,
-          padding: "4px",
+          padding: "0.2rem",
         },
         trigger: {
           fontWeight: "bold",
```

Comment:
lg må ha fontSize 18

```suggestion
          fontWeight: "bold",
          fontSize: "sm",
```